views/models/UsersModel/index.js

```diff
@@ -13,6 +13,24 @@ const UsersModel = {
       isFixed: reader === owner,
     })),
   createReadersList: (readers) => readers.map(({ value }) => value),
+  createUsersAccesList: (access) =>
+    access.map((site) => ({
+      label: site,
+      value: site,
+    })),
+  createUserFromFormValues: (user) => ({
+    ...user,
+    role: user.role?.value || user.role,
```

Comment:
why would `user.role` sometimes be an object with a value key and otherwise not? Can we make the shape consistent?

Reply:
https://github.com/AMP-SCZ/dpdash/commit/1e4e37cd538b8fb695fa0cc8e622ee759d6c5515